### PR TITLE
[EN] corrections for week 09 + correction of "i.e.," in all EN docs

### PR DESCRIPTION
**Pull request by marcozullich**
Fixing typos, missing commas after i.e. and e.g., missing math env, replacing repeated words with synonyms, etc

### CODE REVIEW

File: docs/en/week09/09-1.md [hunk 1]
```diff
@@ -82,12 +82,12 @@ It is a regulariser. The term itself is not trained, it's fixed. It's just the L
 <center><img src="{{site.baseurl}}/images/week09/09-1/AS3giSt.jpg" width="400px" height="200px"/></center>
 **Fig 5:** Invariant Features through Lateral Inhibition
 
-Here, there is a linear decoder with square reconstruction error. There is a criterion in the energy. The matrix S is either determined by hand or learned so as to maximise this term. If the terms in S are positive and large, it implies that the system does not want $z_i$ and $z_j$ to be on at the same time. Thus, it is sort of a mutual inhibition (called natural inhibition in neuroscience). Thus, you try to find a value for S that is as large as possible.
+Here, there is a linear decoder with square reconstruction error. There is a criterion in the energy. The matrix $S$ is either determined by hand or learned so as to maximise this term. If the terms in $S$ are positive and large, it implies that the system does not want $z_i$ and $z_j$ to be on at the same time. Thus, it is sort of a mutual inhibition (called natural inhibition in neuro science). Thus, you try to find a value for $S$ that is as large as possible.
```
Comment (Atcold) [inline docs/en/week09/09-1.md:85]: Nope, *neuroscience* is an actual word.
https://www.oxfordlearnersdictionaries.com/definition/english/neuroscience

File: docs/en/week09/09-2.md [hunk 2]
```diff
@@ -62,7 +62,7 @@ The forward model would like to be able to predict
 
 $$\hat s_{t+1} = g(s_t, a_t, z_t)$$
 
-where $z$ is a latent variable of which we don't know the value. $z$ represents what you cannot know about the world but which still influences the prediction (*i.e.* aleatoric uncertainty). Can regularize $z$ with sparsity, noise, or with an encoder. Can use forward models to learn to plan. The system works by having a decoder decode a concatenation of the state representation and the uncertainty $z$. The best $z$ is defined as the $z$ that minimizes the difference between $\hat s_{t+1}$ and the actual observed $s_{t+1}$.
+where $z$ is a latent variable of which we don't know the value. $z$ represents what you cannot know about the world but which still influences the prediction (*i.e.*, aleatoric uncertainty). You can regularize $z$ with sparsity, noise, or with an encoder. You can use forward models to learn to plan. The system works by having a decoder decode a concatenation of the state representation and the uncertainty $z$. The best $z$ is defined as the $z$ that minimizes the difference between $\hat s_{t+1}$ and the actual observed $s_{t+1}$.
```
Comment (Atcold) [inline docs/en/week09/09-2.md:65]: There is no comma after *i.e.* in (British) English.

File: docs/en/week09/09-3.md [hunk 3]
```diff
@@ -37,14 +37,14 @@ To ensure that high cost is assigned to points outside the data manifold and low
...
+GANs also differ from VAEs through how they produce and use $\vect{z}$. GANs start by sampling $\vect{z}$, similar to the latent space in a VAE. They then use a generative network to map $\vect{z}$ to $\vect{\hat{x}}$. This $\vect{\hat{x}}$ is then sent through a discriminator/cost network to evaluate how "real" it is. One of the main differences from VAE and GAN is that **we do not need to measure a direct relationship (*i.e.*, reconstruction loss) between the output of the generative network $\vect{\hat{x}}$ and real data $\vect{x}$.** Instead, we force $\vect{\hat{x}}$ to be similar to $\vect{x}$ by training the generator to produce $\vect{\hat{x}}$ such that the discriminator/cost network produces scores that are similar to those of real data $\vect{x}$, or more "real".
```
Comment (Atcold) [inline docs/en/week09/09-3.md:47]: Same here as above.

File: docs/en/week09/09-1.md (hunk 1 above)
Comment (marcozullich) [reply docs/en/week09/09-1.md:85]: Interesting, I was sure I did not touch that 🤔

**Comment by Atcold** [comment]
Can you please remove the commas after *i.e.* and *e.g.*, plz?

**Comment by marcozullich** [comment]
Yes, do you want me to do that also in the other files (like 05-2 and 13-3, as in the image I posted) or I'm restricting the corrections to week 09 in this PR and make another PR for the corrections referenced in the image?

**Comment by Atcold** [comment]
I'm not following. What *image* you've posted and where?
Also, it's okay to have a single PR dealing with typos everywhere. I'm not sure I follow the "restriction" suggestion.

**Comment by marcozullich** [comment]
I'm talking about this image higher in the comments

**Comment by Atcold** [comment]
All these should be `*e.g.*` and `*i.e.*`. One more edit, and it's good to go! :D

**Comment by Atcold** [comment]
What's the status here?

File: docs/en/week02/02-2.md [hunk 4]
```diff
@@ -12,7 +12,7 @@ date: 3 Feb 2020
 
 ### Example
 
-We next consider a concrete example of backpropagation assisted by a visual graph. The arbitrary function $G(w)$ is inputted into the cost function $C$, which can be represented as a graph. Through the manipulation of multiplying the Jacobian matrices, we can transform this graph into the graph that will compute the gradients going backwards. (Note that PyTorch and TensorFlow do this automatically for the user, i.e. the forward graph is automatically "reversed" to create the derivative graph that backpropagates the gradient.)
+We next consider a concrete example of backpropagation assisted by a visual graph. The arbitrary function $G(w)$ is inputted into the cost function $C$, which can be represented as a graph. Through the manipulation of multiplying the Jacobian matrices, we can transform this graph into the graph that will compute the gradients going backwards. (Note that PyTorch and TensorFlow do this automatically for the user,*i.e.* the forward graph is automatically "reversed" to create the derivative graph that backpropagates the gradient.)
```
Comment (Atcold) [inline docs/en/week02/02-2.md]:
```suggestion
We next consider a concrete example of backpropagation assisted by a visual graph. The arbitrary function $G(w)$ is inputted into the cost function $C$, which can be represented as a graph. Through the manipulation of multiplying the Jacobian matrices, we can transform this graph into the graph that will compute the gradients going backwards. (Note that PyTorch and TensorFlow do this automatically for the user, *i.e.* the forward graph is automatically "reversed" to create the derivative graph that backpropagates the gradient.)
```

**Comment by Atcold** [comment]
Just awesome! Thank you so much!

**Pull request merged**